cityofaustin/atd-moped
# Add new components to moped-components table

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9795

## Testing
**URL to test:** 
Test locally or deploy to moped-test

**Steps to test:**



![Screen Shot 2022-07-25 at 3 19 13 PM](https://user-images.githubusercontent.com/12474808/180868033-117e4f8c-b026-486c-8ca4-59e0d413422c.png)

In the dropdown, check to see if you can now see Lighting > Street lighting and Lighting > Intersection lighting. Also under Transit > Bus stop. 

Project extent - generic should now be drawn as lines. 

I identified a bug that currently exists in production (https://github.com/cityofaustin/atd-data-tech/issues/9841) where points vs line draw tools are being set by the component type and not subtype. 

---
#### Ship list
